docs/spin1_api/Makefile

```diff
@@ -14,8 +14,8 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 spin1_api.pdf: spin1_api.tex threads.eps
-	latex spin1_api.tex
-	dvipdf spin1_api.dvi
+	pdflatex spin1_api.tex
+	pdflatex spin1_api.tex
```

Comment:
Did you mean to put this call in twice?

Reply:
Yes .Latex sometimes cannot resolve all references in the first pass. The second call gets the reference right.

Sorry to bother you with this nuisance, but I cannot do the review myself.

Reply:
No probs, just wanted to check!